apmpackage/apm/0.1.0/manifest.yml

```diff
@@ -35,12 +35,84 @@ policy_templates:
         title: Secret token
         required: false
         show_user: true
+      - name: api_key_enabled
+        type: bool
+        title: Enable API Key Auth between APM Server and Agents
+        required: false
+        show_user: true
+        default: false
       - name: enable_rum
         type: bool
         title: Enable RUM
         required: true
         show_user: true
         default: false
+      - name: api_key_limit
+        type: int
+        title: Maximum number of allowed API Keys between APM Server and Agents
+        required: false
+        show_user: false
+        default: 100
+      - name: capture_personal_data
+        type: bool
+        title: Capture personal data (e.g. IP, User Agent)
+        required: false
+        show_user: false
+        default: true
+      - name: rum_allow_origins
+        type: string
+        title: RUM - Allow Origins
+        multi: true
+        required: false
+        show_user: false
+        default: ['*']
```

Comment:
filed a bug report https://github.com/elastic/kibana/issues/91401, as this value is currently not allowed.